src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -6547,4 +6547,32 @@ internal sealed partial class LazyTranslatedQueryExpression : BaseTranslatedQuer
         }
         protected override IOperation ExpressionImpl => _lazyExpression.Value;
     }
+
+    internal sealed partial class FlowCapture : Operation, IFlowCaptureOperation
+    {
+        public FlowCapture(int id, SyntaxNode syntax, bool isInitialization, ITypeSymbol type, Optional<object> constantValue) :
+            base(OperationKind.FlowCapture, semanticModel:null, syntax, type, constantValue, isImplicit:true)
```

Comment:
Nit: unconventional formatting on `semanticModel:null` and `isImplicit:true`.

Reply:
> Nit: unconventional formatting on semanticModel:null and isImplicit:true.

I believe this is how IDE formats it.

---
In reply to: [162169363](https://github.com/dotnet/roslyn/pull/24263#discussion_r162169363) [](ancestors = 162169363)